CHANGELOG.md

```diff
@@ -1,3 +1,57 @@
+# 88.2.0
+
+This minor release introduces the Banner component, and a bug fix for Buttons.
```

Comment:
Minor nit: this makes it seems we just started banner. suggested replacement:

"This minor release makes the Banner component generally available, and ..."

Reply:
Done.